client/src/components/LeftSidebar/components/Categorical/components/Category/Category.tsx

```diff
@@ -509,6 +529,14 @@ const CategoryHeader = React.memo(
         </div>
 
         <div className="ignore-capture">
+          {isUserAnnotation ? (
+            <AnnoMenuCategory
```

Comment:
UI callout: `AnnoMenuCategory` is the entry point for rename/delete wired up to the new thunks. Only user annotations render the menu; baked-in categories keep the old read-only rendering.